homework-submission/homework w1/app.js

```diff
@@ -0,0 +1,72 @@
+const ListItem = ({ description, deadLine, className }) => {
+  return <li className={className}>{description + ', ' + deadLine}</li>;
+};
+//ListItem is a reuseable component that has 3 props (description,deadline,className) and return a list item with description and deadLine separated by comma
+class StaticList extends React.Component {
+  render() {
+    return (
+      <ul>
+        Static list
```

Comment:
I don't think it is semantically correct html to place a string like this in your `<ul>`, better to place it above the `<ul>`

Reply:
I don't really know either 
It's modified now